treasury/README.md

```diff
@@ -1,62 +0,0 @@
-## Creating a Treasury Proposal
-
-Your proposal should address a problem, outline a goal, give a detailed account of how you will
```

Comment:
Why delete this?

Reply:
Treasure proposals are conducted on the Darwinia chain, although they are part of the RingDAO. Based on our experience with proposal management, we have a better tool: Tally. It's somewhat difficult for a member to submit a proposal using the Polkadot App. This also increases the maintenance of this repository, as the Polkadot governance guide will change over time. A guide with mistakes is worse than having no guide at all. So, I prefer to keep things simple.

Reply:
Please initial an independent proposal related to this topics.

As this repo related to financial records and external MOUs which need carefully review and audit, I don't think a bottom-up rewrite is good choise.. It will take more time fore me to review it rather than I do it myself.

Please submit a PR for each part, and I will close this PR.